src/main/java/org/geysermc/generator/GenerateNetworkCodec.java

```diff
@@ -0,0 +1,35 @@
+package org.geysermc.generator;
+
+import com.mojang.serialization.DataResult;
+import io.netty.handler.codec.EncoderException;
+import net.minecraft.core.RegistryAccess;
+import net.minecraft.core.RegistrySynchronization;
+import net.minecraft.data.BuiltinRegistries;
+import net.minecraft.nbt.CompoundTag;
+import net.minecraft.nbt.NbtIo;
+import net.minecraft.nbt.NbtOps;
+import net.minecraft.nbt.Tag;
+
+import java.io.File;
+import java.io.IOException;
+
+public class GenerateNetworkCodec {
+    public static void main(String[] args) {
+		Util.initialize();
+        RegistryAccess.Frozen registryAccess = RegistryAccess.fromRegistryOfRegistries(BuiltinRegistries.REGISTRY).freeze();
```

Comment:
This appears to be the proper indent, whilst the rest of the code is not.

Reply:
...how tho